index.js

```diff
@@ -19,7 +19,9 @@ module.exports.pitch = function(remainingRequest) {
 		var chunkNameParam = '';
 	}
 	var result;
-	if(query.lazy) {
+	if(process.env.NODE_ENV !== 'production') {
```

Comment:
I think it's better to introduce an option here, `NODE_ENV === 'production'` is common, but no real standard